pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "lai-textpred"
+authors = [
+    {name = "Lightning-AI", email = "name@lightning.ai"},
```

Comment:
what about the email here?

Reply:
I am not sure if we have any generic one...